src/OpenTK.NT/Native/User32/Structs/RawMouse.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Runtime.InteropServices;
+
+using LONG = System.Int32;
+using ULONG = System.UInt32;
+using USHORT = System.UInt16;
+
+namespace OpenTK.NT.Native
+{
+    /// <summary>
+    /// Contains information about the state of the mouse.
+    /// </summary>
+    [StructLayout(LayoutKind.Explicit)]
+    public struct RawMouse
+    {
+        /// <summary>
+        /// Mouse state. This member can be any reasonable combination of the following.
+        /// MOUSE_ATTRIBUTES_CHANGED
+        /// Mouse attributes changed; application needs to query the mouse attributes.
+        /// MOUSE_MOVE_RELATIVE
+        /// Mouse movement data is relative to the last mouse position.
+        /// MOUSE_MOVE_ABSOLUTE
+        /// Mouse movement data is based on absolute position.
+        /// MOUSE_VIRTUAL_DESKTOP
+        /// Mouse coordinates are mapped to the virtual desktop (for a multiple monitor system).
+        /// </summary>
+        [FieldOffset(0)] public RawMouseFlags Flags; // USHORT in winuser.h, but only INT works -- USHORT returns 0.
```

Comment:
What's up with this implied type mismatch? Is it mismatched?

Reply:
It was mismatched, but I changed it back to ushort.

I can't imagine this just plain not working, I'll take another look at it.